Remotion/Validation/Globalization.UnitTests/LocalizedValidationMessageFactoryTest.cs

```diff
@@ -734,12 +840,17 @@ public void CreateValidationMessageForPropertyValidator_WithDecimalValidator_Ret
       Assert.That(validationMessage, Is.InstanceOf<ResourceManagerBasedValidationMessage>());
 
       Assert.That(validationMessage.ToString(), Does.StartWith("Enter a number with no more than {0} integer digits and {1} decimal places."));
+      Assert.That(((ResourceManagerBasedValidationMessage)validationMessage).ResourceIdentifier,
+          Is.EqualTo(LocalizedValidationMessageFactory.ResourceIdentifier.ValueMustNotExceedDecimalConstraintsValidationMessage));
     }
 
     [Test]
-    [Ignore("TODO RM-5906")]
     public void CreateValidationMessageForObjectValidator_NoLocalizationsAvailableForObjectValidators_ReturnsNull ()
     {
+      var validator = new RealPersonValidator();
+      var validationMessage = _factory.CreateValidationMessageForObjectValidator(validator, TypeAdapter.Create(typeof(Person)));
+
+      Assert.That(validationMessage, Is.Null);
     }
 
     [Test]
```

Comment:
@MichaelKetting Can you please check if this test might be wrong, because the method always returns null and I am not sure if we need to add logic to the implementation or not.

Reply:
The null-value is correct. there is no good way to return a default (meaningful) validation message for generic object validation. So, we return null, which is a valid result for the case that the factory doesn't know what to do.

Reply:
Yes.